mp/src/public/vgui_controls/CvarTextEntry.h

```diff
@@ -34,5 +34,7 @@ namespace vgui
     private:
         ConVarRef m_cvarRef;
         char m_pszStartValue[64];
+
+        char *m_strNumberFormat;
```

Comment:
Store this as a char array, the pointer could be invalidated outside of this class and that would be bad.

```suggestion
        char m_szNumberFormat[8];
```